box.js

```diff
@@ -0,0 +1,16 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// module.exports = { createBox, calculateArea };
```

Comment:
I love that you're including these hints around imports and exports (in the lesson, too). I think it's beneficial to the students!